src/main/java/mate/academy/Main.java

```diff
@@ -1,16 +1,56 @@
 package mate.academy;
 
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import mate.academy.lib.Injector;
+import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
+import mate.academy.model.MovieSession;
+import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
+import mate.academy.service.MovieSessionService;
 
 public class Main {
-    public static void main(String[] args) {
-        MovieService movieService = null;
+    private static final Injector injector =
+            Injector.getInstance("mate.academy");
+    private static MovieService movieService =
+            (MovieService) injector.getInstance(MovieService.class);
+    private static MovieSessionService movieSessionService =
+            (MovieSessionService) injector.getInstance(MovieSessionService.class);
+    private static CinemaHallService cinemaHallService =
+            (CinemaHallService) injector.getInstance(CinemaHallService.class);
 
+    public static void main(String[] args) {
         Movie fastAndFurious = new Movie("Fast and Furious");
         fastAndFurious.setDescription("An action film about street racing, heists, and spies.");
         movieService.add(fastAndFurious);
         System.out.println(movieService.get(fastAndFurious.getId()));
         movieService.getAll().forEach(System.out::println);
+
+        CinemaHall cinemaHall = new CinemaHall();
+        cinemaHall.setCapacity(40);
+        cinemaHall.setDescription("Hall for VIP person");
+        cinemaHallService.add(cinemaHall);
+        cinemaHallService.getAll().forEach(System.out::println);
+
+        MovieSession movieSession1 = new MovieSession();
```

Comment:
```suggestion
        MovieSession firstMovieSession = new MovieSession();
```

it's bad practise to use numbers in variable names

Reply:
done